content/verein/constitution.de.md

```diff
@@ -0,0 +1,174 @@
+{
+    "title": "Satzung",
+    "slug": "satzung",
+    "aliases": ["constitution"],
+    "type": "page"
+}
+
```

Comment:
Since it is now not completely clear what the constitution is for for every user, I think we need some kind of introduction here.

Reply:
Not quite sure what you mean. What has changed? I mean, if you want to write an introduction, feel free but I don't see the need.

Reply:
Before, the constitution was only accessible from the `verein.` webpage, not the general one. So one might stumble upon the constitution without knowing that daterequests.org etc. are run by an association.

But oh well, I will write a short intro.

Reply:
Well yeah, my point is that it was really easy to get there accidentally without realizing. Previously, you probably also clicked a link somewhere and where taken to a page (that to most user) looks exactly like the main page, so… But anyway.